src/components/SearchForm/SearchForm.js

```diff
@@ -1,32 +1,44 @@
-import { useContext, useState } from "react";
+import { useContext, useEffect } from "react";
 import SearchNewsContext from "../../contexts/SearchNewsContext";
-
 import SavedNewsKeywordContext from "../../contexts/SavedNewsKeyword";
+import "./SearchForm.css";
 
 const SearchForm = ({ handleSearchNews }) => {
-  // const [searchInput, setSearchInput] = useState("");
   const { searchInput, setSearchInput } = useContext(SearchNewsContext);
-  const { savedKeyword } = useContext(SavedNewsKeywordContext);
-  console.log("savedKeyword", savedKeyword);
+  const { setSavedKeyword } = useContext(SavedNewsKeywordContext);
+
   const handleSearchInput = (e) => {
     setSearchInput(e.target.value);
   };
   const handleSearchSubmit = (e) => {
     e.preventDefault();
     handleSearchNews(searchInput);
   };
-  // console.log(searchInput);
+
+  useEffect(() => {
+    if (searchInput) {
+      setSavedKeyword(searchInput);
+    } else {
+      setSavedKeyword("");
+    }
+  }, [searchInput, setSavedKeyword]);
 
   return (
     <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component